build.rs

```diff
@@ -1,24 +1,37 @@
-use std::path::{Path, PathBuf};
+use std::path::{PathBuf, Path};
 
 fn main() -> std::io::Result<()> {
     println!("cargo:rerun-if-env-changed=LND_REPO_DIR");
-    let lnd_rpc_dir_owned;
-    let dir = match std::env::var_os("LND_REPO_DIR") {
+    let dir= match std::env::var_os("LND_REPO_DIR") {
         Some(lnd_repo_path) => {
             let mut lnd_rpc_dir = PathBuf::from(lnd_repo_path);
             lnd_rpc_dir.push("lnrpc");
-            lnd_rpc_dir_owned = lnd_rpc_dir;
-            &*lnd_rpc_dir_owned
+            lnd_rpc_dir.to_path_buf()
         },
-        None => Path::new("vendor"),
+        None => Path::new("vendor").to_path_buf(),
```

Comment:
FYI simply `PathBuf::from("vendor")` would work too.

Reply:
there it is! fixed